src/server.ts

```diff
@@ -4,8 +4,6 @@ import 'source-map-support/register'
 import { AbortController } from 'abort-controller'
 Object.assign(global, { AbortController })
 
-import 'ix'
-
```

Comment:
FYI this was in its own "group" so that it doesn't get re-ordered by automatic import ordering. (It's a side effect to add the prototype methods. Could be avoided by using `pipe()` which became much more ergonomic in v5 but was kind of a pain before.)